plan/plan.go

```diff
@@ -43,8 +44,15 @@ func WithNotify(ctx context.Context, in *graph.Graph, notify *graph.Notifier) (*
 	out, err := in.Transform(ctx,
 		notify.Transform(func(id string, out *graph.Graph) error {
 			renderingPlant.Graph = out
+
 			pipeline := Pipeline(out, id, renderingPlant)
-			val, pipelineErr := pipeline.Exec(out.Get(id))
```

Comment:
As with apply- does it make sense to pass the entire node in here and handle it correctly inside the pipeline?